app/assets/stylesheets/mw-runtime-base.scss

```diff
@@ -438,10 +438,7 @@ div.intra-page-nav {
   .question {
     margin: 20px;
     display: inline-block;
-    // the questions sit in two columns in the full width layout.
-    // The 438px width is computed to maximize their size:
-    // 960/2 = 480, 480 - margins = 440, then there is 2 px of padding somewhere
-    width: 438px;
+    width: 45%;
```

Comment:
I changed this to match what works in the CC Runtime base theme. 

In MW Runtime base themes there were line breaks between what should've been side-by-side boxes in full-width layout mode. I tried to track down the cause, but could only get as far as seeing that removing one pixel of width from one of the boxes fixed the problem. There wasn't an obvious reason why and I couldn't come up with an intuitive way to accommodate it.

Using a width of 45% instead of 438px removes about 5 pixels in width from each box. That doesn't seem like much to me, but if it's a problem, I can try to spend more time figuring out where that extra pixel comes from and maybe get the widths back to 438px.